debian/rules

```diff
@@ -21,26 +25,31 @@ override_dh_auto_build:
 	go build -ldflags='-extldflags -Wl,-soname,pam_aad.so' -buildmode=c-shared -o pam_aad.so ./pam
 
 	# Build NSS library
-	go build -ldflags='-extldflags -Wl,-soname,libnss_aad.so.2' -buildmode=c-shared -o libnss_aad.so.2 ./nss
+	make -C nss/
 
 	# Build the CLI
 	# DH_GOLANG_BUILDPKG=github.com/ubuntu/aad-auth/cmd/aad-auth dh_auto_build
 
-override_dh_auto_install:
-	dh_auto_install -- --no-source
+	# Build the NSS helper binary
+	go build ./nss/aad-auth
 
+override_dh_auto_install:
 	# Install PAM module configuration
 	mkdir -p debian/tmp/usr/share/pam-configs
 	cp debian/libpam-aad.pam-auth-update debian/tmp/usr/share/pam-configs/aad
 
-	# Comment configuration and install it to /etc
+	# Install configuration template to /etc
 	mkdir -p debian/tmp/etc
-	sed -e 's/^\(\)/# \1/' conf/aad.conf.template > debian/tmp/etc/aad.conf
+	cp conf/aad.conf.template debian/tmp/etc/aad.conf
 
 	# Install PAM and NSS libraries
 	mkdir -p debian/tmp/usr/lib/$(DEB_HOST_GNU_TYPE)/security
 	cp pam_aad.so debian/tmp/usr/lib/$(DEB_HOST_GNU_TYPE)/security
-	cp libnss_aad.so.2 debian/tmp/usr/lib/$(DEB_HOST_GNU_TYPE)
+	cp nss/libnss_aad.so.2 debian/tmp/usr/lib/$(DEB_HOST_GNU_TYPE)
```

Comment:
We should use DESTDIR so that make install install it in the correct directory.

Reply:
yes, taking DESTDIR to be as standard as possible.

Reply:
No, this looks good to me. Would have been great to have some more dh_ calls here, but I guess anyway that for this frankeinstein project, we will have a different approach with Rust :)